web/src/js/routes/EmergencyRoute.jsx

```diff
@@ -2,12 +2,12 @@ import _ from 'lodash';
 import React from 'react';
 import ReactMarkdown from 'react-markdown';
 import { connect } from 'react-redux';
-import { Link } from 'react-router';
+import { Link } from 'react-router-dom';
 import shallowCompare from 'react-addons-shallow-compare';
 
-import Body from 'components/Body';
-import EmergencyHotline from 'components/EmergencyHotline';
-import emergencyKyr from 'static/emergency-kyr';
+import Body from '../components/Body';
```

Comment:
Can you change these back to the absolute paths? I aliased them with webpack for convenience but also because there's a tool called madge that will show your import graphs if the paths are absolute.